src/core/database/query/lists.ts

```diff
@@ -23,3 +33,34 @@ export function calculateTotalAndPaginateList(
 
   return query.asResult<{ items: string[]; total: number }>();
 }
+
+export const defaultSorter: Sorter = (
+  q,
+  sortInput,
+  order,
+  securedProperties,
+  sortValInput
+) => {
+  //The properties that are stored as strings
+  const stringProperties = ['name'];
+  const sortInputIsString = stringProperties.includes(sortInput);
+
+  //if the sortInput, e.g. name, is a string type, check to see if a custom sortVal is given.  If not, coerse the default prop.value to lower case in the orderBy clause
+  const sortValSecuredProp =
+    sortValInput || (sortInputIsString ? 'toLower(prop.value)' : 'prop.value');
+
+  const sortValBaseNodeProp = sortInputIsString
+    ? `toLower(node.${sortInput})`
+    : `node.${sortInput}`;
+
+  return sortInput in securedProperties
+    ? q
+        .match([
+          node('node'),
+          relation('out', '', sortInput, { active: true }),
+          node('prop', 'Property'),
+        ])
+        .with('*')
+        .orderBy(sortValSecuredProp, order)
+    : q.with('*').orderBy(sortValBaseNodeProp, order);
```

Comment:
The issue with this is last orderBy statement is if we sort by something that doesn't match any property values we can all results returned unsorted. ( e.g. sort by "asdf") This exists on master on most list queries

Reply:
I'd really like to define these sort keys in a stricter way. There's also this though https://github.com/SeedCompany/cord-api-v3/issues/1363#issuecomment-712307415

Reply:
Is any specific action you'd want me to take for this ticket?